kaggle/api/kaggle_api_extended.py

```diff
@@ -397,9 +397,12 @@ class KaggleApi:
     config_values: Dict[str, str] = {}
     already_printed_version_warning = False
 
-    args: List[str] = []  # DEBUG Add --local to use localhost
+    args: List[str] = []
     if os.environ.get('KAGGLE_API_ENVIRONMENT') == 'LOCALHOST':
+        # Make it verbose when running in the debugger.
         args = ['--verbose', '--local']
+    elif os.environ.get('KAGGLE_API_ENDPOINT') == 'http://localhost':
+        args = ['--local']
```

Comment:
Why is one verbose and the other is not? Should it be:

```suggestion
    if os.environ.get('KAGGLE_API_ENVIRONMENT') == 'LOCALHOST' or os.environ.get('KAGGLE_API_ENDPOINT') == 'http://localhost':
        # Make it verbose when running in the debugger.
        args = ['--verbose', '--local']
```

Reply:
There are two ways to run the tests. `hatch run test` from the command line and a Rider run config. It's easier to run from the CLI, so I opted to not make it verbose. When I'm running a single test or the whole script in the debugger I like to have it verbose. (I actually wrote it your way originally, then realized the two cases were quite different is actual use.)